kartothek/core/docs.py

```diff
@@ -146,6 +146,12 @@
     "partition_size": """
     partition_size: int
         Amount of metapartitions to cluster into one dask partition.
+        Note: Do not pass this argument and `n_partitions` together.
```

Comment:
Maybe we can put a link to the actual Dask docs in here since that probably explains the intention of the parameters best. 
Ideally we'd be using [intersphinx](http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) but as a first step a plain link would suffice

Reply:
I had copied the docstrings from dask.

Do you want me to provide the link to the docstrings in code-base or dask documentation?